package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Infrastructure as SQL",
   "main": "dist/index.js",
   "scripts": {
-    "build": "yarn clean && tsc",
+    "build": "yarn clean && tsc && copyfiles -u 1 src/**/*.sql dist/",
```

Comment:
does this copy it into `dist/src/**/*.sql` or just `dist/*.sql`?

Reply:
In the exact same directory structure as it was in before. :) I found out about it in a blog post with someone needing something similar for a web build process but not wanting to switch to babel.